install.sh

```diff
@@ -172,11 +172,15 @@ install_nvm_as_script() {
   nvm_download -s "$NVM_SOURCE_LOCAL" -o "$INSTALL_DIR/nvm.sh" || {
     echo >&2 "Failed to download '$NVM_SOURCE_LOCAL'"
     return 1
-  }
+  } &
   nvm_download -s "$NVM_EXEC_SOURCE" -o "$INSTALL_DIR/nvm-exec" || {
     echo >&2 "Failed to download '$NVM_EXEC_SOURCE'"
     return 2
-  }
+  } &
+  for job in $(jobs -p | sort)
```

Comment:
will this include only the two previous jobs? or could user jobs show up here too?